extensions/github-review-requests/src/actionablePullRequests.tsx

```diff
@@ -1,60 +1,132 @@
-import { Color, MenuBarExtra, open } from "@raycast/api";
+import { MenuBarExtra, open, Icon, Color } from "@raycast/api";
+import { useMemo } from "react";
 import usePulls from "./hooks/usePulls";
+import { groupedByAttribute } from "./util";
+import { PullRequestShort, ReviewDecision } from "./types";
 import PullRequestItem from "./components/PullRequestItem";
-import { PullRequestShort } from "./types";
+
+interface PullRequestGroup {
+  [owner: string]: PullRequestShort[];
+}
 
 const actionablePullRequests = () => {
-  const { isLoading, login, updatedPulls, recentlyVisitedPulls, Refresh, visitPull } = usePulls();
+  const { isLoading, isReady, login, updatedPulls, recentlyVisitedPulls, visitPull, runPullIteration } = usePulls();
 
-  const title = getTitle(updatedPulls);
-  const icon = getIcon(updatedPulls);
+  const title = useMemo(() => (updatedPulls.length > 0 ? `${updatedPulls.length}` : "🎉"), [updatedPulls]);
 
-  const showSeparator = updatedPulls.length > 0 && recentlyVisitedPulls.length > 0;
-  const forceRefreshSeparator = updatedPulls.length > 0 || recentlyVisitedPulls.length > 0;
+  const pullsByOwner = useMemo(
+    () => groupedByAttribute(updatedPulls, "owner.login") as PullRequestGroup,
+    [updatedPulls]
+  );
 
   return (
-    <MenuBarExtra isLoading={isLoading} icon={icon} title={title}>
-      {updatedPulls.length > 0 &&
-        updatedPulls.map((pull, index) => (
-          <PullRequestItem
-            key={pull.id}
-            pull={pull}
-            index={index}
-            showMyIcon
-            onAction={() => open(pull.url).then(() => visitPull(login, pull))}
-          />
-        ))}
-
-      {updatedPulls.length === 0 && recentlyVisitedPulls.length === 0 && !isLoading && (
-        <MenuBarExtra.Item title={`No updated pulls at this time`} />
+    <MenuBarExtra isLoading={isLoading} icon="icon.png" title={title} tooltip="Your Pull Requests">
+      {updatedPulls.length === 0 && (
+        <MenuBarExtra.Section>
+          <MenuBarExtra.Item title="🧹 No upcoming PRs." />
+        </MenuBarExtra.Section>
       )}
-
-      {showSeparator && <MenuBarExtra.Separator />}
-
-      {recentlyVisitedPulls.length > 0 && (
-        <MenuBarExtra.Submenu title="Recently Visited">
+      <MenuBarExtra.Section>
+        <MenuBarExtra.Submenu title="Recent" icon={Icon.Clock}>
           {recentlyVisitedPulls.map(pull => (
             <PullRequestItem key={pull.id} pull={pull} onAction={() => open(pull.url)} />
           ))}
         </MenuBarExtra.Submenu>
-      )}
+      </MenuBarExtra.Section>
+      <>
+        {Object.entries(pullsByOwner).map(([owner, pulls]) => {
+          if (pulls.length === 0) {
+            return null;
+          }
 
-      {forceRefreshSeparator && <MenuBarExtra.Separator />}
+          // Pulls
+          const wfmPulls = pulls.filter(
```

Comment:
Not a fan of "wfm", "wfc", "wfr", "nrr". It takes five minutes to forget what that is. It's best to give it a proper name